src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -75,6 +82,49 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
     this.setState({ showCreatePipelineRunModal });
   };
 
+  handleChange = event => {
+    const inputValue = event.target.value;
+    this.setState({
+      currentFilterValue: inputValue
+    });
+  };
+
+  handleAddFilter = () => {
```

Comment:
We could add a small bit of input validation here too, if I don't put a `:` in my query then display something (a small error icon in the textfield?) suggesting that only key:value pairs are accepted here